src/main/webapp/gameDialogue.js

```diff
@@ -92,3 +93,14 @@ function modifyIfFinalStage() {
         }
       });
 }
+
+//play button functions
+function playbutton() {
+  if(document.getElementById('player-button').innerText == "PAUSE"){
+    document.getElementById('player-button').innerText = "PLAY";
```

Comment:
Change this to "play music" so it's clear that it's different than the "play" button that started the game